lib/addons/p5.dom.js

```diff
@@ -1118,7 +1118,7 @@
         cb = arguments[i];
       }
     }
-    if (navigator.getUserMedia) {
+    if (navigator.getUserMedia || navigator.mediaDevices.getUserMedia) {
```

Comment:
Since the code below is using `navigator.mediaDevices.getUserMedia` exclusively, it might make sense to skip the currently used check entirely and check for `navigator.mediaDevices && navigator.mediaDevices.getUserMedia` only?

Reply:
That makes sense to me! I left the old check in there in case of older browsers. But if it's just going to use mediaDevices anyway, then there's no point in doing the check for navigator.getUserMedia